src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AddAzureSqlDatabaseToFailoverGroup.cs

```diff
@@ -25,34 +25,34 @@ namespace Microsoft.Azure.Commands.Sql.FailoverGroup.Cmdlet
     /// <summary>
     /// Cmdlet to add Azure Sql Databases into a Failover Group
     /// </summary>
-    [Cmdlet(VerbsCommon.Add, "AzureRmSqlDatabaseToFailoverGroup", SupportsShouldProcess = true,
+    [Cmdlet(VerbsCommon.Add, "AzureRmSqlDatabaseToFailoverGroup",
         ConfirmImpact = ConfirmImpact.Medium)]
     public class AddAzureSqlDatabaseToFailoverGroup : AzureSqlFailoverGroupCmdletBase
     {
         /// <summary>
         /// The name of the Azure SQL Database FailoverGroup
         /// </summary>
         [Parameter(Mandatory = true,
-            ValueFromPipeline = true,
+            ValueFromPipelineByPropertyName = true,
             Position = 2,
-            HelpMessage = "The name of the Azure SQL Failover Group.")]
+            HelpMessage = "The name of the Azure SQL Database Failover Group.")]
         [ValidateNotNullOrEmpty]
         public string FailoverGroupName { get; set; }
 
         /// <summary>
-        /// The Azure SQL Databases to be added to the secondary server
+        /// The Azure SQL Database to be added to the secondary server
         /// </summary>
         [Parameter(Mandatory = true,
             ValueFromPipeline = true,
-            HelpMessage = "The Azure SQL Databases to be added to the secondary server.")]
+            HelpMessage = "The Azure SQL Database to be added to the secondary server.")]
```

Comment:
That was what I thought but the databases are actually being piped in and they will be executed one by one through batch jobs. So a singular would make a little more sense here